diff --git a/components/device/silabs/si91x/mcu/drivers/service/nvm3/component/sl_si91x_common_flash_nvm3.slcc b/components/device/silabs/si91x/mcu/drivers/service/nvm3/component/sl_si91x_common_flash_nvm3.slcc
index e7286a8d..ccbce202 100644
--- a/components/device/silabs/si91x/mcu/drivers/service/nvm3/component/sl_si91x_common_flash_nvm3.slcc
+++ b/components/device/silabs/si91x/mcu/drivers/service/nvm3/component/sl_si91x_common_flash_nvm3.slcc
@@ -27,9 +27,9 @@ requires:
   - name: wiseconnect3_common
   - name: wifi
   - name: sl_si91x_mcu
-  - name: sl_si91x_internal_stack
-  - name: sl_si91x_basic_buffers
-  - name: wifi_resources
+#  - name: sl_si91x_internal_stack
+#  - name: sl_si91x_basic_buffers
+#  - name: wifi_resources
   - name: freertos
   - name: device_init_nvic
 include:
diff --git a/components/service/network_manager/src/sl_net_for_lwip.c b/components/service/network_manager/src/sl_net_for_lwip.c
index f5d23d37..d5fcd585 100644
--- a/components/service/network_manager/src/sl_net_for_lwip.c
+++ b/components/service/network_manager/src/sl_net_for_lwip.c
@@ -264,17 +264,17 @@ static void set_sta_link_up(sl_net_wifi_client_profile_t *profile)
     }
   }
 
-#if LWIP_IPV4
-  uint64_t addr = wifi_client_context->netif.ip_addr.addr;
-#endif
-#if LWIP_IPV6
-  uint64_t addr = wifi_client_context->netif.ip_addr.u_addr.ip4.addr;
-#endif
-  SL_DEBUG_LOG("DHCP IP: %u.%u.%u.%u\n",
-               NETIF_IPV4_ADDRESS(addr, 0),
-               NETIF_IPV4_ADDRESS(addr, 1),
-               NETIF_IPV4_ADDRESS(addr, 2),
-               NETIF_IPV4_ADDRESS(addr, 3));
+// #if LWIP_IPV4
+//   uint64_t addr = wifi_client_context->netif.ip_addr.addr;
+// #endif
+// #if LWIP_IPV6
+//   uint64_t addr = wifi_client_context->netif.ip_addr.u_addr.ip4.addr;
+// #endif
+//   SL_DEBUG_LOG("DHCP IP: %u.%u.%u.%u\n",
+//                NETIF_IPV4_ADDRESS(addr, 0),
+//                NETIF_IPV4_ADDRESS(addr, 1),
+//                NETIF_IPV4_ADDRESS(addr, 2),
+//                NETIF_IPV4_ADDRESS(addr, 3));
 
 #if LWIP_IPV6
 #if LWIP_IPV6_AUTOCONFIG
@@ -384,19 +384,19 @@ sl_status_t sl_net_wifi_client_up(sl_net_interface_t interface, sl_net_profile_i
   VERIFY_STATUS_AND_RETURN(status);
 
   set_sta_link_up(&profile);
-  uint64_t addr                        = wifi_client_context->netif.ip_addr.addr;
+  uint64_t addr                        = wifi_client_context->netif.ip_addr.u_addr.ip4.addr;
   profile.ip.ip.v4.ip_address.bytes[0] = NETIF_IPV4_ADDRESS(addr, 0);
   profile.ip.ip.v4.ip_address.bytes[1] = NETIF_IPV4_ADDRESS(addr, 1);
   profile.ip.ip.v4.ip_address.bytes[2] = NETIF_IPV4_ADDRESS(addr, 2);
   profile.ip.ip.v4.ip_address.bytes[3] = NETIF_IPV4_ADDRESS(addr, 3);
 
-  addr                              = wifi_client_context->netif.gw.addr;
+  addr                              = wifi_client_context->netif.gw.u_addr.ip4.addr;
   profile.ip.ip.v4.gateway.bytes[0] = NETIF_IPV4_ADDRESS(addr, 0);
   profile.ip.ip.v4.gateway.bytes[1] = NETIF_IPV4_ADDRESS(addr, 1);
   profile.ip.ip.v4.gateway.bytes[2] = NETIF_IPV4_ADDRESS(addr, 2);
   profile.ip.ip.v4.gateway.bytes[3] = NETIF_IPV4_ADDRESS(addr, 3);
 
-  addr                              = wifi_client_context->netif.netmask.addr;
+  addr                              = wifi_client_context->netif.netmask.u_addr.ip4.addr;
   profile.ip.ip.v4.netmask.bytes[0] = NETIF_IPV4_ADDRESS(addr, 0);
   profile.ip.ip.v4.netmask.bytes[1] = NETIF_IPV4_ADDRESS(addr, 1);
   profile.ip.ip.v4.netmask.bytes[2] = NETIF_IPV4_ADDRESS(addr, 2);
